TripalCultivate/TripalCultivate-Theme
# Ensures that the header looks good on our docker

This PR makes the site look like the below screenshot when you run our docker :-) 

Specifically,
- Updates the docker to use the `notheme` version of the Tripal Cultivate Base docker image to inherit from (added via https://github.com/TripalCultivate/TripalCultivate/pull/11)
- Sets the TripalCultivate logo as the default logo for this theme
- Updates configuration for the system branding block to enable display of the slogan
- updates the theme settings to use the default logo by default
- set the site name to make it more obvious you are working in the theme docker ;-p

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/1566301/ed6500b2-bbff-4404-8f24-bbdd242539e9)

## Testing

1. Build a fresh docker on this branch and start a container.
```
docker build --build-arg drupalversion=10.2.x-dev --build-arg phpversion=8.3 --tag=trpcultivate-theme:G0.7 ./
docker run --publish=80:80 -tid --name=themeG0.7 trpcultivate-theme:G0.7
```
2. Check out the front page. It should look like the above screenshot :-)
